src/VisualStudio/TestUtilities/IntegrationLog.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public sealed class IntegrationLog : IDisposable
+    {
+        private static readonly string ProcDumpFilePath = IntegrationHelper.LocateFile("procdump.exe", "https://live.sysinternals.com/procdump.exe");
+        private static IntegrationLog _current;
+
+        private long _scenarioStart;
+        private long _testStart;
+        private string _dumpsFolder;
+        private string _outputFolder;
+        private string _screenshotsFolder; // TODO: Provide Screenshot Functionality
```

Comment:
Please file a bug for the TODO and reference it here or if the comment is unneeded delete it.